README.md

```diff
@@ -1 +1,274 @@
-# Mobile-First
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [MOBILEFIRST] <a name="about-project"></a>
+
+
+Make sure that there are no linter errors:
+    -Linter should be set up correctly for this repository.
+    -Linter result should be green.
+    -There should be no custom changes to linter config.
+    -Make sure that you used correct GitHub Flow:
+    -README file should be descriptive.
+    -Commit messages should be meaningful.
+    -Pull Request should have a descriptive title explaining what changes are inside.
+    -Pull Request should have a short summary describing introduced changes in more detail.
+    -Make sure that you documented your work in a professional way:
+    -Branch name should describe a feature or action (e.g. 'created_sidebar' or 'added_comments_section').
+    -Pull Request is made from 'feature branch' into the main branch.
+    ##HTML/CSS requirements
+    Follow our list of best practices for HTML & CSS.
+    Project requirements
+    For this project, you should use the HTML&CSS linter
+
+The 2 sections you need to build for this project are:
+
+the toolbar (or header):
+Toolbar
+
+the headline section (right after the header):
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
```

Comment:
[OPTIONAL] Please delete all the placeholders and replace them with your project info. These are the instructions for you, not for the user.